selendroid-standalone/src/main/java/io/selendroid/standalone/android/impl/AbstractDevice.java

```diff
@@ -596,4 +597,8 @@ public boolean equals(Object o) {
   public int hashCode() {
     return device.hashCode();
   }
+
+  public String getDeviceType() {
+    return deviceType;
```

Comment:
1. Since <code>selendroid</code> already used <code>device</code> for for both emulator and real hardware as object under test.
2. <code>model</code> is only used in the limitation of real hardware object

Therefore I think <code>deviceType</code> is a logical choice to specify a different type of device to test if it is is an emulator, or a different type of model if it is a hardware device. Other than that, there is no special reasons to use <code>deviceType</code> in place of <code>model</code>.